mappings/MdxAdOverlayState.mapping

```diff
@@ -0,0 +1 @@
+CLASS rbk MdxAdOverlayState
```

Comment:
Do you know what `Mdx` stands for? I'd rather not add mappings we ourselves don't understand the naming for

Reply:
I believe that [MDX] (https://mdxjs.com/) is markdown with JSX components..

Reply:
Cool, didn't know about this project yet :) Although I doubt this is what the name references, does the app really parse Markdown and JavaScript? Maybe we should ask in the ReVanced Discord

Reply:
Searching for mdx in classes: I found com.google.android.apps.youtube.app.mdx.tvsignin and other classes referring to tv...
So it might have to do with casting?

Reply:
Let's leave it as `Mdx`, since that's actually a package name. If we ever find out its meaning, we can still change it